transformer_engine/pytorch/csrc/common.h

```diff
@@ -168,12 +168,13 @@ class Float8BlockQuantizer : public Quantizer {
   // Which float8 type is used for q data.
   DType dtype;
 
- private:
```

Comment:
It's a bit suspect how we're exposing the member variables publicly.

Reply:
We need to access `force_pow_2_scales` and `amax_epsilon` in https://github.com/NVIDIA/TransformerEngine/blob/59cb49c4cfdfc56fba52fcc617ccc8b15f1a9a0b/transformer_engine/pytorch/csrc/extensions/cast.cpp#L77-L78

Changed it to public members as in `Float8CurrentScalingQuantizer`.